content/en/tracing/runtime_metrics/go.md

```diff
@@ -27,28 +27,29 @@ tracer.Start(tracer.WithRuntimeMetrics())
 
 View runtime metrics in correlation with your Go services on the [Service page][1] in Datadog.
 
-By default, runtime metrics from your application are sent every 10 seconds to the Datadog Agent with DogStatsD over port `8125`. Make sure that [DogStatsD is enabled for the Agent][2]. If your Datadog Agent DogStatsD address differs from the default `localhost:8125`, use the `WithDogstatsdAddress` option or the environment variables `DD_AGENT_HOST` and `DD_DOGSTATSD_PORT`.
+By default, runtime metrics from your application are sent every 10 seconds to the Datadog Agent with DogStatsD. Make sure that [DogStatsD is enabled for the Agent][2]. If your Datadog Agent DogStatsD address differs from the default `localhost:8125`, use the `WithDogstatsdAddress` option or the environment variables `DD_AGENT_HOST` and `DD_DOGSTATSD_PORT`. See the `WithDogstatsdAddress` documentation for information on how to configure the connection to [DogStatsD][3]
```

Comment:
What about specifying the version directly, aka 
>use the `WithDogstatsdAddress` (available starting in 1.18.0)

so the API docs don't need to be referenced?

Reply:
Done.